Sources/Public/Item Views/Shape.swift

```diff
@@ -0,0 +1,24 @@
+// Created by Bryan Keller on 9/12/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import CoreGraphics
+
+/// Represents a background or highlight layer shape; used by `DayView` and `DayOfWeekView`.
+public enum Shape: Hashable {
+  case circle
+  case roundRect(cornerRadius: CGFloat)
```

Comment:
```suggestion
  case rectangle(cornerRadius: CGFloat)
```
since it's not always a rounded rect

Reply:
Renamed to `square` since day / day-of-the-week views are always 1:1 aspect ratio.

Reply:
Seems like a requirement that might change independent of this API though.

Reply:
That's true. Alright, back to `rectangle` it is!